IEEE-PSH/hackpsh-web
# Simple Announcement Prototype

# Description

Basic Form (composed of a title and content) input for now, just to ensure stability and negligent performance impacts. Rich Text Editor will come down the road as a future improvement.

# Testing Guideline

Ensure that only admins can traverse to /announcements/create-post, and announcements visually appear on the /announcements page and can be accessed by only users who have completed onboarding and are authenticated (signed-in).
